File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/WebhooksUpdateWebhooksRequest.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class WebhooksUpdateWebhooksRequest {
    @JsonProperty("errorResponseJsonTemplate")
    public String errorResponseJsonTemplate;
    @JsonProperty("facilityLicenseNumbers")
    public String facilityLicenseNumbers;
    @JsonProperty("objectType")
    public String objectType;
    @JsonProperty("serverPublicKeyFingerprint")
    public String serverPublicKeyFingerprint;
    @JsonProperty("status")
    public String status;
    @JsonProperty("template")
    public String template;
    @JsonProperty("tpiApiKey")
    public String tpiApiKey;
    @JsonProperty("url")
    public String url;
    @JsonProperty("userApiKey")
    public String userApiKey;
    @JsonProperty("verb")
    public String verb;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/WebhooksUpdateWebhooksRequest.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class WebhooksUpdateWebhooksRequest(
    @JsonProperty("errorResponseJsonTemplate")
    val errorResponseJsonTemplate: String? = null,
    @JsonProperty("facilityLicenseNumbers")
    val facilityLicenseNumbers: String? = null,
    @JsonProperty("objectType")
    val objectType: String? = null,
    @JsonProperty("serverPublicKeyFingerprint")
    val serverPublicKeyFingerprint: String? = null,
    @JsonProperty("status")
    val status: String? = null,
    @JsonProperty("template")
    val template: String? = null,
    @JsonProperty("tpiApiKey")
    val tpiApiKey: String? = null,
    @JsonProperty("url")
    val url: String? = null,
    @JsonProperty("userApiKey")
    val userApiKey: String? = null,
    @JsonProperty("verb")
    val verb: String? = null
)
